geode-core/src/main/java/org/apache/geode/internal/logging/LogWriterFactory.java

```diff
@@ -87,19 +87,19 @@ public static InternalLogWriter createLogWriterLogger(final boolean isLoner,
         // LOG:CONFIG:
         logger.info(LogMarker.CONFIG, Banner.getString(null));
       }
+      System.setProperty(InternalLocator.INHIBIT_DM_BANNER, "true"); // Ensure no more banners will
```

Comment:
Yes, I had a suspicion that this would be an issue - do you have any suggestions concerning how to do this without changing a system property? I was considering just checking the log for whether it contains the banner, but this seems inefficient.